waffle/models.py

```diff
@@ -396,7 +399,8 @@ def _get_group_ids(self) -> set[Any]:
 
     def is_active_for_user(self, user: AbstractBaseUser) -> bool | None:
         is_active = super().is_active_for_user(user)
-        if is_active:
+
+        if is_active is True or is_active is False:
```

Comment:
same here

Reply:
this is more readable

Reply:
would `if is_active is not None` work the same? it is also more succinct and clear